src/utils/config.ts

```diff
@@ -22,28 +19,28 @@ import { NonEmptyString } from "italia-ts-commons/lib/strings";
 
 export const RedisParams = t.intersection([
   t.interface({
-    REDIS_URL: NonEmptyString
+    REDIS_URL: NonEmptyString,
   }),
   t.partial({
     REDIS_CLUSTER_ENABLED: t.boolean,
     REDIS_PASSWORD: NonEmptyString,
     REDIS_PORT: NonEmptyString,
-    REDIS_TLS_ENABLED: t.boolean
-  })
+    REDIS_TLS_ENABLED: t.boolean,
+  }),
 ]);
 export type RedisParams = t.TypeOf<typeof RedisParams>;
 
 export const ContactPersonParams = t.intersection([
   t.interface({
     COMPANY_EMAIL: EmailString,
-    COMPANY_FISCAL_CODE: OrganizationFiscalCode,
+    COMPANY_FISCAL_CODE: WithinRangeString(8, 16),
```

Comment:
Since this definition is intended for SPIS SP' s contact person we can safely revert this file, we can leave `OrganizationFiscalCode` without modifications.

Reply:
done in https://github.com/pagopa/hub-spid-login-ms/pull/40/commits/d45c6dd848ee37216f1fa4d12004c0463c1a5693